### ISSUE TITLE
[Issue] [ResizableTextArea] doesn't work properly when [ReadOnly] is applied and result is different if applied before or after the [ResizableTextArea] Attribute 

### ISSUE THREAD

**Issue by MostHated** [dbrizov/NaughtyAttributes#180]
Hey there,
    I was trying to create a [ReadOnly] [ResizableTextArea] to use as a full width slim label, which could also give me a place to apply a header, but I noticed that it was acting strange when I tried.

When applied like this below:
```cs 
[ResizableTextArea] 
[ReadOnly]
public string info;
```
The result is this:
![](https://i.imgur.com/jYWQ3l0.png)
The text is not ReadOnly, but also, the multiline part of the textbox does not work. The text just continues out of frame. If you enter a newline, the cursor moves to the new line, but the box remains the same size so the above line of text is no longer visible unless you move the cursor with the up arrow to the prior line.

If the ReadOnly tag is applied like this:
```cs
[ReadOnly]
[ResizableTextArea] 
public string info;
```
Then the ReadOnly aspect works, but ResizableTextArea is not applied.
![](https://i.imgur.com/Cwf5e7L.png)

Is it possible to have the ResizableTextArea's full width, but also keep the ReadOnly aspect of it, so I can populate the text myself? Even better would be to not show the fieldname label at all and just use the text within the box as the "label".

That said, the reason I came across this was I wanted to add the Header text to the top of an inspector, but the only things that are currently there either can't have another attribute on them without interfering with the current ones, or doesn't/won't display it because of the type that it is (ReorderableList that is within the FoldOut Attribute didn't let me apply a Header to it, it simply didn't show). 

Is there some sort of way or field type that can be added, but the field itself is not visible at all but the effect that the attribute would provide still shows? I know you can use [HideInInspector] but that also hides the Attribute effect, such as a Header, or a HorizontalLine. Is there any way to keep the Header/Line but hide the field completely, that it needs to be attached to? 

Something like:
```cs
[Header("Standalone Header")]
[HideField]
private bool noShow;
```
and then the bool itself is not in the inspector, but the Header only will be displayed.

**Comment by dbrizov** [comment]
Ok, I know why [ReadOnly] and [ResizableTextArea] don't work together. This is because they are both drawer attributes. I fix this by making the [ReadOnly] attribute a meta attribute and not an actual drawer.

**Comment by MostHated** [comment]
I appreciate you looking into this and the recommendation. Using the [EnableIf] attribute definitely sounds like a good workaround, I will give that a try.

**Comment by dbrizov** [comment]
Fixed in ce8eaf2658f37a1e89efedf6111df9b02f238cfd
